### PR TITLE
🔥 🔥 🔥 Add About, Rules, Schedule, Onsite Contest, and many things!

### PR DESCRIPTION
**Pull request by alijaya**

### CODE REVIEW

File: onsite.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+| Saturday, 9 May 2020 | 09.00 | Opening ceremony |
+|                      | 11.00 | Practice session |
+|                      | 13.00 | Lunch |
+|                      | 14.00 | Small excursion |
+|                      | 18.00 | Dinner |
+|                      | 20.00 | Free time |
+| Sunday, 10 May 2020  | 08.00 | Onsite competition |
+|                      | 13.00 | Lunch |
+|                      | 14.00 | Excursion |
+|                      | 18.00 | Dinner and closing ceremony (awading) |
```
Comment (prabowo02) [inline onsite.md]:
```suggestion
|                      | 18.00 | Dinner and closing ceremony (awarding) |
```

File: about.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+layout: single-page-md
+title: About APIO 2020
+key: about
+---
+
+The Asia-Pacific Informatics Olympiad (APIO) is an IOI-like competition for delegations within the South Asian / Western Pacific region. The first competition was organized by the Australian IOI team in 2007.
```
Comment (prabowo02) [inline about.md]:
```suggestion
The Asia-Pacific Informatics Olympiad (APIO) is an IOI-like competition for delegations within the Asian and Western Pacific regions. The first competition was organized by the Australian IOI team in 2007.
```

ngikutin yang call-for-tasks.html

File: about.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+APIO 2020 will be hosted by Indonesia. Usually the contest will be held online.
```
Comment (prabowo02) [inline about.md]:
```suggestion
APIO 2020 will be hosted by Indonesia. Usually, the contest will be held online.
```

File: about.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+But in APIO 2020, for the first time, in addition to online version, there will be held also onsite contest in Indonesia.
```
Comment (prabowo02) [inline about.md]:
```suggestion
But in APIO 2020, for the first time, in addition to the online version, there will be held also onsite contest in Indonesia.
```

File: about.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+As usual, in the online version, the participants will be competing at the contest sites within their own coutries or areas.
```
Comment (prabowo02) [inline about.md]:
```suggestion
As usual, in the online version, the participants will be competing at the contest sites within their own countries or areas.
```

File: about.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+And in addition to that, there will be several countries that will be invited to send their delegation to Indonesia to participate in the onsite contest.
+
+The eligible participants are the one that is also eligible for the IOI 2020. The leaders of each eligible country to participate will register their participants for the APIO 2020. If you are a student wishing to compete, please contact your local informatics olympiad organization.
+
+APIO 2020 is organized by *IA TOKI*, and the online session has the window from Friday, 8 May 2020 (UTC+0 06.00) to Sunday, 10 May 2020 (UTC+0 06.00).
+And the onsite event will be held from Friday, 8 May 2020 to Monday, 11 May 2020.
+
+IA TOKI is an association of the former medalist in Indonesian National Olympiad in Informatics.
+It consists of the university students, professional, and educator.
```
Comment (prabowo02) [inline about.md]:
```suggestion
It consists of university students, professionals, and educators.
```

File: about.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+The Asia-Pacific Informatics Olympiad (APIO) is an IOI-like competition for delegations within the South Asian / Western Pacific region. The first competition was organized by the Australian IOI team in 2007.
+
+APIO 2020 will be hosted by Indonesia. Usually the contest will be held online.
+But in APIO 2020, for the first time, in addition to online version, there will be held also onsite contest in Indonesia.
+
+As usual, in the online version, the participants will be competing at the contest sites within their own coutries or areas.
+
+And in addition to that, there will be several countries that will be invited to send their delegation to Indonesia to participate in the onsite contest.
```
Comment (prabowo02) [inline about.md]:
```suggestion
And in addition to that, there will be several countries that will be invited to send their delegations to Indonesia to participate in the onsite contest.
```

File: onsite.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+---
+layout: single-page-md
+title: APIO 2020 Onsite
+key: onsite
+---
+
+### Registration for Onsite Contest
+* There will be only 10 that are invited and 6 participants for each countries. The committee will accept the confirmation of the country that want to do it onsite in First Come First Serve base.
```
Comment (prabowo02) [inline onsite.md]:
```suggestion
* There will be only 10 that are invited and 6 participants for each country. The committee will accept the confirmation of the country that wants to do it onsite in the First Come First Serve base.
```

File: onsite.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+* To register your country, please refer first to the general [registration page](registration).
+* In the confirmation phase, the leader need to state their confirmation to participate in onsite contest.
```
Comment (prabowo02) [inline onsite.md]:
```suggestion
* In the confirmation phase, the leader needs to state their confirmation to participate in the onsite contest.
```

File: onsite.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+* In the registration phase, the leader need to state the participants that will participate in the onsite contest.
```
Comment (prabowo02) [inline onsite.md]:
```suggestion
* In the registration phase, the leader needs to state the participants that will participate in the onsite contest.
```

File: onsite.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+### Onsite Contest Information
+* The APIO 2020 onsite have the same rule as the [online contest format](rules#online-contest-format), with several differences.
+* The participants of each invited country will take the contest in Jakarta, Indonesia.
+* The contest will be held on Sunday, 10 May 2020 (UTC+7 08.00) for 5 hours.
+* Beside the contest day, there will be a time for excursion and visiting popular places in Jakarta.
```
Comment (prabowo02) [inline onsite.md]:
```suggestion
* Besides the contest day, there will be a time for excursion and visiting popular places in Jakarta.
```

File: onsite.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+* There are no differences between the onsite participants and the online participants. The winners will be decided equally as if the onsite participants are the online participants.
+* If onsite participants [win medals](rules#medals-distribution), they will be awarded physical medals in closing ceremony / awarding night.
```
Comment (prabowo02) [inline onsite.md]:
```suggestion
* If onsite participants [win medals](rules#medals-distribution), they will be awarded physical medals in the closing ceremony / awarding night.
```

File: onsite.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+
+### Onsite Contest Schedule
+
+Draft schedule, subject to change! For the general schedule, check [schedule page](schedule).
```
Comment (prabowo02) [inline onsite.md]:
```suggestion
Draft schedule, subject to change! For the general schedule, check [the schedule page](schedule).
```

File: rules.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
+---
+layout: single-page-md
+title: APIO 2020 Rules
+key: rules
+---
+
+### Online Contest Format
+* The APIO 2020 online contest is a 5 hour contest with flexible start time within determined window.
```
Comment (prabowo02) [inline rules.md]:
```suggestion
* The APIO 2020 online contest is a 5-hour contest with flexible start time within a determined window.
```

File: rules.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+* The contest will be held between Friday, 8 May 2020 (UTC+0 06.00) and Sunday, 10 May 2020 (UTC+0 06.00).
+* Each country must run one or more official contest sites.
+* All participants must take the contest at an official contest site
+* Each contest site starts the contest at the time of its own choice within the window. Different sites from within the same country may also start the contest at different time.
```
Comment (prabowo02) [inline rules.md]:
```suggestion
* Each contest site starts the contest at the time of its own choice within the window. Different sites from within the same country may also start the contest at different times.
```

**Comment by alijaya** [comment]
done
nunggu ashar ngemerge

File: about.md [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+
+The eligible participants are the one that is also eligible for the IOI 2020. The leaders of each eligible country to participate will register their participants for the APIO 2020. If you are a student wishing to compete, please contact your local informatics olympiad organization.
+
+APIO 2020 is organized by *IA TOKI*, and the online session has the window from Friday, 8 May 2020 (UTC+0 06.00) to Sunday, 10 May 2020 (UTC+0 06.00).
+And the onsite event will be held from Friday, 8 May 2020 to Monday, 11 May 2020.
+
+IA TOKI is an association of the former medalist in Indonesian National Olympiad in Informatics.
```
Comment (fushar) [inline about.md]:
```suggestion
IA TOKI is an association of former medalists of Indonesian National Olympiad in Informatics.
```

File: about.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+The Asia-Pacific Informatics Olympiad (APIO) is an IOI-like competition for delegations within the South Asian / Western Pacific region. The first competition was organized by the Australian IOI team in 2007.
+
+APIO 2020 will be hosted by Indonesia. Usually the contest will be held online.
+But in APIO 2020, for the first time, in addition to online version, there will be held also onsite contest in Indonesia.
+
+As usual, in the online version, the participants will be competing at the contest sites within their own coutries or areas.
+
+And in addition to that, there will be several countries that will be invited to send their delegation to Indonesia to participate in the onsite contest.
+
+The eligible participants are the one that is also eligible for the IOI 2020. The leaders of each eligible country to participate will register their participants for the APIO 2020. If you are a student wishing to compete, please contact your local informatics olympiad organization.
```
Comment (fushar) [inline about.md]:
```suggestion
The eligible participants are the ones that are also eligible for the IOI 2020. The leaders of each eligible country to participate will register their participants for the APIO 2020. If you are a student wishing to compete, please contact your local informatics olympiad organization.
```

File: rules.md [hunk 18]
```diff
@@ -0,0 +1,21 @@
...
+* Each contest site must be fully supervised in order to guarantee that the contest rules be observed by all the students.
+* Each country may consist of up to 60 students. However, only the top six students will form the official team, according to the contest results.
+* Students are eligible for this contest if and only if they are eligible for the IOI 2019.
```
Comment (fushar) [inline rules.md]:
```suggestion
* Students are eligible for this contest if and only if they are eligible for IOI 2020.
```

File: rules.md [hunk 19]
```diff
@@ -0,0 +1,21 @@
...
+layout: single-page-md
+title: APIO 2020 Rules
+key: rules
+---
+
+### Online Contest Format
+* The APIO 2020 online contest is a 5 hour contest with flexible start time within determined window.
+* The contest will be held between Friday, 8 May 2020 (UTC+0 06.00) and Sunday, 10 May 2020 (UTC+0 06.00).
+* Each country must run one or more official contest sites.
+* All participants must take the contest at an official contest site
```
Comment (fushar) [inline rules.md]:
```suggestion
* All participants must take the contest at an official contest site.
```

File: schedule.md [hunk 20]
```diff
@@ -0,0 +1,12 @@
+---
+layout: single-page-md
+title: APIO 2020 Schedule
+key: schedule
+---
+
+| Activity | Timeline (in UTC+0) |
+|----------|----------|
+| [Deadline Leader Confirmation](registration) | Friday, 13 March 2020 |
```
Comment (fushar) [inline schedule.md]:
```suggestion
| [Leader Confirmation Deadline](registration) | Friday, 13 March 2020 |
```

File: schedule.md [hunk 21]
```diff
@@ -0,0 +1,12 @@
...
+| [Deadline Task Submission](call-for-tasks) | Sunday, 15 March 2020 |
```
Comment (fushar) [inline schedule.md]:
```suggestion
| [Task Submission Deadline](call-for-tasks) | Sunday, 15 March 2020 |
```

File: schedule.md [hunk 22]
```diff
@@ -0,0 +1,12 @@
...
+| [Deadline Participant List Registration](registration) | Friday, 10 April 2020 |
```
Comment (fushar) [inline schedule.md]:
```suggestion
| [Participant List Registration Deadline](registration) | Friday, 10 April 2020 |
```

File: registration.md [hunk 23]
```diff
@@ -0,0 +1,55 @@
+---
+layout: single-page-md
+title: APIO 2020 Registration
+key: registration
+---
+
+### Confirmation Phase
+The leader of each country listed below will be contacted by [email](infoapio2020@iatoki.id).
```
Comment (fushar) [inline registration.md]:
```suggestion
The leader of each country listed below will be contacted by [email](info.apio2020@toki.id).
```

File: _sass/_status.scss [hunk 24]
```diff
@@ -0,0 +1,62 @@
...
+}
+.status-np {
+  @extend .status;
+
+  background: purple;
+
+  &::before {
+    content: "Not Participating";
+  }
+}
```
Comment (jonathanirvings) [inline _sass/_status.scss]: last newline?

File: about.md [hunk 25]
```diff
@@ -0,0 +1,22 @@
+---
+layout: single-page-md
+title: About APIO 2020
+key: about
+---
+
+The Asia-Pacific Informatics Olympiad (APIO) is an IOI-like competition for delegations within the Asian and Western Pacific regions. The first competition was organized by the Australian IOI team in 2007.
+
+APIO 2020 will be hosted by Indonesia. Usually, the contest will be held online.
```
Comment (jonathanirvings) [inline about.md]: `s/Usually, the contest will be held online./In the past, APIO contests were held online.`

File: about.md [hunk 26]
```diff
@@ -0,0 +1,22 @@
...
+But in APIO 2020, for the first time, in addition to the online version, there will be held also onsite contest in Indonesia.
+
+As usual, in the online version, the participants will be competing at the contest sites within their own countries or areas.
+
+And in addition to that, there will be several countries that will be invited to send their delegations to Indonesia to participate in the onsite contest.
```
Comment (jonathanirvings) [inline about.md]: Remove `And` and combine this with the previous paragraph?

File: about.md [hunk 27]
```diff
@@ -0,0 +1,22 @@
...
+
+The eligible participants are the ones that are also eligible for the IOI 2020. The leaders of each eligible country to participate will register their participants for the APIO 2020. If you are a student wishing to compete, please contact your local informatics olympiad organization.
+
+APIO 2020 is organized by *IA TOKI*, and the online session has the window from Friday, 8 May 2020 (UTC+0 06.00) to Sunday, 10 May 2020 (UTC+0 06.00).
+And the onsite event will be held from Friday, 8 May 2020 to Monday, 11 May 2020.
```
Comment (jonathanirvings) [inline about.md]: Remove `And`

File: onsite.md [hunk 28]
```diff
@@ -0,0 +1,40 @@
+---
+layout: single-page-md
+title: APIO 2020 Onsite
+key: onsite
+---
+
+### Registration for Onsite Contest
+* There will be only 10 that are invited and 6 participants for each country. The committee will accept the confirmation of the country that wants to do it onsite in the First Come First Serve basis.
```
Comment (jonathanirvings) [inline onsite.md]: `s/10/10 countries`?

Comment (jonathanirvings) [inline onsite.md]: `s/wants to do it onsite/wants to participate in the onsite contest`

File: onsite.md [hunk 29]
```diff
@@ -0,0 +1,40 @@
...
+* To register your country, please refer first to the general [registration page](registration).
+* In the confirmation phase, the leader needs to state their confirmation to participate in onsite contest.
```
Comment (jonathanirvings) [inline onsite.md]: `s/in onsite/in the onsite`

File: onsite.md [hunk 30]
```diff
@@ -0,0 +1,40 @@
...
+* The participants of each invited country will take the contest in Jakarta, Indonesia.
+* The contest will be held on Sunday, 10 May 2020 (UTC+7 08.00) for 5 hours.
+* Besides the contest day, there will be a time for excursion and visiting popular places in Jakarta.
+* There are no differences between the onsite participants and the online participants. The winners will be decided equally as if the onsite participants are the online participants.
+* If onsite participants [win medals](rules#medals-distribution), they will be awarded physical medals in the closing ceremony / awarding night.
+
+
+### Onsite Contest Schedule
+
+Draft schedule, subject to change! For the general schedule, check [the schedule page](schedule).
```
Comment (jonathanirvings) [inline onsite.md]: `s/Draft schedule, subject to change/Plesae note that the following is a draft schedule and subject to change.`

File: registration.md [hunk 31]
```diff
@@ -0,0 +1,55 @@
+---
+layout: single-page-md
+title: APIO 2020 Registration
+key: registration
+---
+
+### Confirmation Phase
+The leader of each country listed below will be contacted by [email](info.apio2020@iatoki.id).
```
Comment (jonathanirvings) [inline registration.md]: `mailto:info.apio2020@iatoki.id`

File: registration.md [hunk 32]
```diff
@@ -0,0 +1,55 @@
...
+Then they can reply with their confirmation of joining the competition.
+In addition to that they can opt-in to onsite competition in Indonesia too, please refer to [onsite page](onsite#registration-for-onsite-contest).
```
Comment (jonathanirvings) [inline registration.md]: put a comma after `to that`?

Comment (jonathanirvings) [inline registration.md]: `s/to onsite/to the onsite`?

File: schedule.md [hunk 33]
```diff
@@ -0,0 +1,12 @@
...
+title: APIO 2020 Schedule
+key: schedule
+---
+
+| Activity | Timeline (in UTC+0) |
+|----------|----------|
+| [Leader Confirmation Deadline](registration) | Friday, 13 March 2020 |
+| [Task Submission Deadline](call-for-tasks) | Sunday, 15 March 2020 |
+| [Participant List Registration Deadline](registration) | Friday, 10 April 2020 |
+| [Online Competition Window](rules) | Friday, 8 May 2020, 06.00 - Sunday, 10 May 2020, 06.00 |
```
Comment (jonathanirvings) [inline schedule.md]: Put `(UTC+0)` after each `06.00`

File: _includes/sidebar.html [hunk 34]
```diff
@@ -9,13 +9,61 @@
         <span class="pt-icon-standard pt-icon-feed sidebar-icon"></span>
         <a href="/">News</a>
       </div>
+      <div
```
Comment (jonathanirvings) [inline _includes/sidebar.html]: You can add news about these additional pages in `index.html`, something like:

```
<div class="pt-card card__title">
    <h3 class="card__title__text">APIO 2020 registration, schedule, rules, and onsite competition</h3>
    <p class="card__title__right">6 February 2020</p>
  </div>
  <div class="pt-card card__content">
    <p>
      We have added several pages in the website, including registration, schedule, rules, and onsite competition. Please do check them.
    </p>
  </div>
```

Put the link to the page for each of the `registration`, `schedule`, `rules`, and `onsite competition` in the post.

File: schedule.md [hunk 35]
```diff
@@ -0,0 +1,12 @@
+---
+layout: single-page-md
+title: APIO 2020 Schedule
+key: schedule
+---
+
```
Comment (jonathanirvings) [inline schedule.md]: You can also write
```
The schedule is not exhaustive yet.
The schedule for practice session, task translation, and result announcement is yet to be announced.
```
here.

File: schedule.md (hunk 33 above)
Comment (alijaya) [reply schedule.md]: udah di header table nya di state itu UTC+0

Comment (jonathanirvings) [reply schedule.md]: Oh right. Ok.

File: _sass/_status.scss (hunk 24 above)
Comment (jonathanirvings) [reply _sass/_status.scss]: seems like the last newline is still missing? unresolving this.

Comment (alijaya) [reply _sass/_status.scss]: Wut... Perasaan udah... ._. bentar gua cek lagi

**Pull request merged**